libraries/exoplayer/src/main/java/androidx/media3/exoplayer/drm/DefaultDrmSession.java

```diff
@@ -699,6 +719,7 @@ private boolean maybeRetryRequest(Message originalMsg, MediaDrmCallbackException
         // The error is fatal.
         return false;
       }
+      currentKeyLoadInfo.addRetryLoadRequest(loadEventInfo);
       synchronized (this) {
```

Comment:
This might be useful if you want the trails of `LoadEventInfo`'s including redirects in the path.  The main `LoadEventInfo` should include all the time.

Reply:
see last comment, with that redirects will be reported correctly.